### PR TITLE
Add Roblox OAuth2 Provider for Laravel Socialite

### PR DESCRIPTION
**Pull request by sympact06**
This commit introduces the Roblox OAuth2 Provider for Laravel Socialite. It includes the provider's installation and usage instructions in the README file as well as the implementation of the Provider class. It also updates the monorepo-builder.yml file to include a new reference to the Roblox provider repository.

### CODE REVIEW

File: src/Roblox/Provider.php [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+            'username' => $user['preferred_username'], // Roblox username (not display name)
+            'nickname' => $user['preferred_username'], // Roblox display name (not guaranteed to be unique)
+            'picture'   => $user['picture'] ?? null,  // Roblox may leave this null if the account is new
+        ]);
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public static function additionalConfigKeys()
```
Comment (atymic) [inline src/Roblox/Provider.php:86]: Can leave this out

File: src/Roblox/composer.json [hunk 2]
```diff
@@ -0,0 +1,32 @@
+{
+    "name": "socialiteproviders/discord",
```
Comment (atymic) [inline src/Roblox/composer.json:2]: this is wrong

**Review by atymic — CHANGES_REQUESTED** [review]

File: src/Roblox/Provider.php (hunk 1 above)
Comment (sympact06) [reply src/Roblox/Provider.php]: Fixed

**Comment by sympact06** [comment]
Fixed the comments

File: src/Roblox/Provider.php [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+        );
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function getCodeFields($state = null)
+    {
+        return array_merge(parent::getCodeFields($state), [
+            'client_id' => $this->clientId,
```
Comment (lucasmichot) [inline src/Roblox/Provider.php:40]: Aren't these properties already set in the parent method?

**Review by atymic — APPROVED** [review]

**Comment by atymic** [comment]
Please PR to website https://github.com/SocialiteProviders/website

**Pull request merged**